tests/test_validators.py

```diff
@@ -42,6 +44,18 @@ def test_invalid_url(self):
             url = 'https://hasgeek'
             self.form.process(url=url)
             assert not self.form.validate()
+            
+    def test_valid_emoji(self):
+        with self.app.test_request_context('/'):
+            dat = u'\u2604'
```

Comment:
You can also try giving an actual emoji as input to see if it works, like `😱`.

Reply:
Added! Seems to work.